pkg/git/git.go

```diff
@@ -274,9 +274,17 @@ func CloneOrOpenRepo(repoPath, repoURL string, useSSH bool) (*Repo, error) {
 		targetDir = t
 	}
 
+	var progress *os.File
```

Comment:
I wonder if we should pipe to stderr on normal log-level so we can dump progress logs in case of error